public/index.html

```diff
@@ -7,6 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="COVID-19 Open Movement in Columbus" />
+  <meta http-equiv="refresh" content="0;URL='https://www.cantstopcolumbus.com/'" />
```

Comment:
any concerns with the comment from [w3schools](https://www.w3schools.com/tags/att_meta_http_equiv.asp) about the refresh tag breaking [W3C's Web Content Accessibility Guidelines](http://www.w3.org/WAI/intro/wcag.php).